code/modules/mob/living/carbon/human/examine.dm

```diff
@@ -194,27 +194,44 @@
 				msg += "[t_He] [t_has] minor [dna.species.brute_damage_desc].\n"
 			else if(temp < 50)
 				msg += "[t_He] [t_has] <b>moderate</b> [dna.species.brute_damage_desc]!\n"
-			else
+			else if(temp < 90)
 				msg += "<B>[t_He] [t_has] severe [dna.species.brute_damage_desc]!</B>\n"
+			else
+				msg += "<B>[t_He] [t_has] extreme [dna.species.brute_damage_desc]!</B>\n"
 
 		temp = getFireLoss()
 		if(temp)
 			if(temp < 25)
-				msg += "[t_He] [t_has] minor [dna.species.burn_damage_desc].\n"
+				msg += "<span class='engradio'>[t_He] [t_has] minor [dna.species.burn_damage_desc].</span>\n"
 			else if (temp < 50)
-				msg += "[t_He] [t_has] <b>moderate</b> [dna.species.burn_damage_desc]!\n"
+				msg += "<span class='engradio'>[t_He] [t_has] <b>moderate</b> [dna.species.burn_damage_desc]!</span>\n"
+			else if(temp < 90)
+				msg += "<span class='engradio'><B>[t_He] [t_has] severe [dna.species.burn_damage_desc]!</B></span>\n"
 			else
-				msg += "<B>[t_He] [t_has] severe [dna.species.burn_damage_desc]!</B>\n"
+				msg += "<span class='engradio'><B>[t_He] [t_has] extreme [dna.species.burn_damage_desc]!</B></span>\n"
 
 		temp = getCloneLoss()
 		if(temp)
 			if(temp < 25)
-				msg += "[t_He] [t_has] minor [dna.species.cellular_damage_desc].\n"
+				msg += "<span class='deadsay'>[t_He] [t_has] minor [dna.species.cellular_damage_desc].</span>\n"
 			else if(temp < 50)
-				msg += "[t_He] [t_has] <b>moderate</b> [dna.species.cellular_damage_desc]!\n"
+				msg += "<span class='deadsay'>[t_He] [t_has] <b>moderate</b> [dna.species.cellular_damage_desc]!</span>\n"
+			else if(temp < 90)
+				msg += "<span class='deadsay'><b>[t_He] [t_has] severe [dna.species.cellular_damage_desc]!</b></span>\n"
 			else
-				msg += "<b>[t_He] [t_has] severe [dna.species.cellular_damage_desc]!</b>\n"
+				msg += "<span class='deadsay'><b>[t_He] [t_has] extreme [dna.species.cellular_damage_desc]!</b></span>\n"
 
+		temp = getToxLoss()
+		if(temp)
+			if(temp >= 90 && !obscured)
+				msg += "<span class='green'><b>[t_He] look[p_s()] deathly ill!</b></span>\n"
+
+		temp = getOxyLoss()
+		if(temp)
+			if((temp >= 30 && temp <=120) && (!appears_dead && !obscured))
+				msg += "<span class='medradio'>[p_theyre(TRUE)] gasping for air!</span>\n"
```

Comment:
```suggestion
				msg += "<span class='medradio'>[t_He] is blue in the face!</span>\n"
```

Oxyloss could be from a few sources, so maybe this is the most source-agnostic method expressing this?